### ISSUE TITLE
Quickgrid shows three loading dots with the loading placeholder 

### ISSUE THREAD

**Issue by CrahunGit** [dotnet/aspnetcore#49440]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

If you set the loading render fragment component will show your custom html followed by … because its set on css content property.

### Expected Behavior

It should show only your set render fragment 

### .NET Version

.net 8 preview 6

**Comment by schaveyt** [comment]
@CrahunGit How do you even show the loading render fragment? I looked in the source and I couldnt find anything 😦 . 

I am using the ItemsProvider to pull remote data and it take a bit of time to fetch. I cannot determine when the QuickGrid is down loading....

@danroth27 ccing.

**Comment by schaveyt** [comment]
So I found the `PlaceholderTemplate` but when using the ItemsProvider it never seems to be disappear.

**Comment by dariatiurina** [comment]
@CrahunGit Thank you for reaching us! I have started investigation this issue and wanted to ask if this issue concerns the PlaceholderTemplate in QuickGrid component with the virtualization enabled? If not, please answer with the code that reproduces this issue. Thanks!

**Comment by CrahunGit** [comment]
Yes, it’s in virtualization mode, while it’s loading. I saw that this elipsis is in css
